web/content/docs/tools/meshing/vtu2grid/index.pandoc

```diff
@@ -17,8 +17,8 @@ The tool is part of the official [OpenGeoSys git repository](https://github.com/
 ## Usage
 
 ```bash
-   Vtu2Grid.exe  -i <input.vtu> -o <output.vtu> -x <floating point number>
-                 [-y <floating point number>] [-z <floating point number>]
+   Vtu2Grid  -i <input.vtu> -o <output.vtu> -x <floating point number>
```

Comment:
Same mismatch in other new documentation snippets; Sometimes `.exe` sometimes `linux`.